content/en/user-guide/tools/cloud-pods/remotes/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Cloud Pod Remotes"
+weight: 5
+categories: ["LocalStack Pro", "Tools", "Persistence", "CLI"]
+description: >
+  Guide to the usage of Cloud Pods remotes
+aliases:
+  - /tools/cloud-pods/remotes/
+---
+
+{{< alert title="Beta">}}
+Cloud Pod Remotes is a feature that is currently in beta.
+{{< /alert >}}
+
+A remote is the place where your Cloud Pods are stored.
+By default, we store the Cloud Pods' artifacts in the LocalStack platform.
+However, for various reasons, e.g., data regulation and sovereignty, you might want to store Cloud Pods in a different location that is under your control.
+
+We currently offer two alternative remotes:
+- S3 bucket remote storage;
+- [ORAS](https://oras.land/) (OCI Registry as Storage) remote storage.
+
+Our CLI has been expanded and now offers commands to create and delete remotes:
+
+```bash
+localstack pod remote --help
+Usage: localstack pod remote [OPTIONS] COMMAND [ARGS]...
+
+  Manage cloud pod remotes
+
+Options:
+  -h, --help  Show this message and exit.
+
+Commands:
+  add     Add a remote
+  delete  Delete a remote
+```
+
+Moreover, the `localstack pod remotes` command will show all the registered remotes.
```

Comment:
nit: we briefly discussed this in the past - should be good to merge as-is for now, 👍  but maybe we can unify this a bit over time (at least we could add a `pod remote list` command, and see how the `remotes` command shortcut compares to other parts of the CLI)

Reply:
100%. Opened a [small PR](https://github.com/localstack/localstack-ext/pull/2138). We could take the change to introduce it into 2.3. What do you say?